README.md

```diff
@@ -141,38 +144,29 @@ class PlaidEventContainer extends React.Component {
     }
   }
 
-  render() {
-    return (
-      <PlaidLink
-        token={<#GENERATED_LINK_TOKEN#>}
-        onSuccess={data => console.log('success: ', data)}
-        onExit={data => console.log('exit: ', data)}
-      >
-        <Text>Add Account</Text>
-      </PlaidLink>
-    );
-  }
+  ...
 }
 ```
 
 You can also use the `usePlaidEmitter` hook in react functional components:
 
 ```javascript
-  usePlaidEmitter((event) => {
+  usePlaidEmitter((event: LinkEvent) => {
     console.log(event)
   })
 ```
 
 ### Customizing the PlaidLink component
 
-By default, `PlaidLink` renders a `TouchableOpacity` component. You may override the component used by passing `component` that accepts an `onPress` callback prop and `componentProps`. For example:
+`PlaidLink` wraps the view you provide as a child with a `Pressable` component and intercepts the `onPress` event. For example:
 
 ```jsx
 <PlaidLink
-  token = {"<#GENERATED_LINK_TOKEN#>"}
-  component= {Button}
-  componentProps = {{title: 'Add Account'}}
-  onSuccess = {(result) => {console.log('Success: ', result)}}
-  onError = {(result) => {console.log('Error: ', result)}}
->
+  tokenConfig={{
```

Comment:
Does this example still make sense?
Maybe better to remove it altogether.

e.g. there used to be a `component` and `componentProps` attribute, but now you just have to pass a `Pressable` child. (which isn't even in the example below 😅 )